src/main/java/com/joy/joyfillandroid/MultipleChoice.kt

```diff
@@ -0,0 +1,100 @@
+package com.joy.joyfillandroid
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.GradientDrawable
+import android.os.Build
+import android.util.AttributeSet
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.annotation.RequiresApi
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.joy.joyfillandroid.adapter.MutlipleChoiceAdapter
+import com.joy.joyfillandroid.model.MultiPleChoiceModel
+
+@RequiresApi(Build.VERSION_CODES.P)
+class MultipleChoice @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int =0
+) : LinearLayout(context, attrs, defStyleAttr){
+
+    private val multiSelection = false
+    private val multipleChoiceDspMode ="readonly"
+
+    private val textColor ="#121417"
+    private val white ="#FFFFFF"
+    private val borderGrey="#D1D1D6"
+    private val blue = "#256FFF"
+    private val containerBorderColor="#D1D1D6"
+
+    var list= ArrayList<MultiPleChoiceModel>()
+    private lateinit var adapter: MutlipleChoiceAdapter
+
+    init {
+        orientation = VERTICAL
+        val layoutParams = LinearLayout.LayoutParams(
+            LinearLayout.LayoutParams.MATCH_PARENT,
+            dpToPx(190)
+        )
+        layoutParams.setMargins(dpToPx(10),dpToPx(10),dpToPx(10),dpToPx(10))
+        setLayoutParams(layoutParams)
+
+        val label = TextView(context)
+        label.text = "MultiChoice Field"
```

Comment:
Shouldn't this label be dynamic or is this label for the button that is used to trigger the view for seeing this component?